project/Build.scala

```diff
@@ -430,4 +431,25 @@ object ScaldingBuild extends Build {
     )
     }
   )
+
+  lazy val executionTutorial = Project(
```

Comment:
We don't have a project for tutorial yet. Not sure if that's designed. I created this seprate project for execution only, because in the example I want to show how to run the job as a pure scala/java application. Therefore I need the com.twitter.scalding.tutorial package to be built into the jar.